tests/io/MemoryHandleTest.php

```diff
@@ -16,6 +16,13 @@
 
 // @oss-disable: <<Oncalls('hack')>>
 final class MemoryHandleTest extends HackTest {
+  public async function testWriteOutOfBound(): Awaitable<void> {
+    $handle = new IO\MemoryHandle('f');
+
+    await $handle->writeAllAsync('Hello, World!');
```

Comment:
Thanks - can you add some expectations about the contents of `$handle->getBuffer()` after each of these writes please?

Reply:
done